tests/zfs-tests/tests/functional/cli_root/zpool_scrub/zpool_scrub_date_range_001.ksh

```diff
@@ -0,0 +1,93 @@
+#!/bin/ksh -p
+# SPDX-License-Identifier: CDDL-1.0
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or https://opensource.org/licenses/CDDL-1.0.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+# Copyright 2025 Klara, Inc.
+# Copyright 2025 Mariusz Zaborski <oshogbo@FreeBSD.org>
+#
+
+. $STF_SUITE/include/libtest.shlib
+. $STF_SUITE/tests/functional/cli_root/zpool_scrub/zpool_scrub.cfg
+
+#
+# DESCRIPTION:
+#       Verify that the date range scrub only scrubs the files that were
+#       created/modified within a given time slot.
+#
+# STRATEGY:
+#     1. Write a file.
+#     2. Force a sync of everything via export/import.
+#     3. Wait for one minute.
+#     4. Repeat steps 1, 2, and 3 four two times.
+#     5. Inject checksum errors into all 3 files.
+#     6. Scrub the date range for the first file.
+#     7. Verify that the first file is scrubbed.
+#     8. Verify that newer files are not scrubbed.
+#     9. Repeat steps 6–8 for each of the remaining 2 files.
+#
+
+verify_runnable "global"
+
+function cleanup
+{
+	log_must zinject -c all
+	rm -f $TESTDIR/*_file
+	log_must restore_tunable SPA_NOTE_TXG_TIME
+}
+
+log_onexit cleanup
+
+log_assert "Verifiy scrub, -E, and -S show expected status."
+
+log_must save_tunable SPA_NOTE_TXG_TIME
+log_must set_tunable64 SPA_NOTE_TXG_TIME 30
+
+typeset -a date_list
+for i in `seq 0 2`; do
+	log_must sleep 60
+	log_must zpool export $TESTPOOL
+	log_must zpool import $TESTPOOL
```

Comment:
Is there are reason `sync_pool $TESTPOOL true` couldn't be used here, and below, instead of a full import/export cycle?